kuma/users/templates/account/email.html

```diff
@@ -8,7 +8,7 @@
 
     {% if user.emailaddress_set.all().exists() %}
 
-      <p>{{ _('{email} is your <em>primary</em> email address')|fe(email=user.email) }}</p>
+      <p>{{ _('%(email)s is your <em>primary</em> email address'|safe, email=user.email) }}</p>
```

Comment:
After further digging it seems like with newstyle gettext calls and autoescape=True, jinja2 marks msgid's as safe itself.